packages/mobile/ios/Podfile

```diff
@@ -47,7 +47,8 @@ target 'AudiusReactNative' do
    react_native_post_install(
       installer,
       config[:reactNativePath],
-      :mac_catalyst_enabled => false
+      :mac_catalyst_enabled => false,
+      # :ccache_enabled => true
```

Comment:
is this supposed to be commented?

Reply:
yeah weirdly is what's in the transition doc